index.js

```diff
@@ -52,7 +51,7 @@ function checkGuess() {
     numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
+  if (attempts === maxNumberOfAttempts) {
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
```

Comment:
Just below here on line 59 you set `guessInput.value = ''`. Something to consider here is that you are guessing `numbers` and not strings and setting your default number value to an empty string may cause a bug in your application.

For example, in your app, without adding any value to the input. Click your "Submit Guess" button. You'll see that it shows your guess as `NaN`. This means "Not a Number". This is a value that should never be displayed to a user. It's important to try to keep your types the same when working with inputs unless you have a specific use case for not doing so.

There are a lot of easy ways to fix this, and it all depends on your preference. Off the top of my head, here are a couple things you can do:

1. Just set the default to 0 in your setup function, and reset it to 0 after each incorrect guess. (EX: `guessInput.value = 0`)
2. If the value is <= 0, you could disable the button (as long as the random number could never be 0 or a negative number).

Happy to talk more about "data types" if you're curious about why it's important.